next/src/components/sections/ArticlesSection/ArticlesFiltered.tsx

```diff
@@ -72,14 +72,21 @@ const ArticlesFiltered = ({ section }: Props) => {
     queryKey: getArticlesQueryKey(filters, locale),
     queryFn: () => articlesFetcher(filters, locale),
     placeholderData: keepPreviousData,
+    // don't fetch if this section contains only manually selected articles
+    enabled: !!category || tags.length > 0 || adminGroupDocumentIds.length > 0,
   })
 
+  const articlesToShow = [...(articles?.filter(isDefined) ?? []), ...(data?.hits ?? [])].slice(
```

Comment:
Should't we filter out pinned articles from data.hits? Otherwise they will show twice.